1.8/translations/translations.md

```diff
@@ -29,3 +29,17 @@ If these exist, they should be uploaded to `images/<language name>/` and `groupi
 #### Finishing Touches
 
 Go to **User CP > Edit Options**. You should see the translation in the **Other Options** menu. Try out the new translation and make sure that everything works.
+
+### Default forum translation
+
+If you want your translation to be default on forum so when users and guests access it you can change it by going to **Admin CP > Configuration > Settings > General Configuration > Default Language**.
+That will make all new users and guests to see board firstly on that translation.
+
+**Please note:** Changing this in Admin CP will not force new default translation to existing users on board. They need to change it manually in their profiles.
+
+#### Default translation for Admin CP
+
+You can make default Admin CP translation to be different than default board translation.
```

Comment:
You can make default Admin CP translation to be different than default board translation.

Should be imo:

You can set the default Admin CP language to be different than default forums translation.